doc/examples/time_domain_nfchoa.py

```diff
@@ -0,0 +1,58 @@
+"""
+Create some examples in the time domain.
+
+Simulate and plot impulse behavior for NFC-HOA.
+
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+import sfs
+
+# simulation parameters
+fs = 44100
+grid = sfs.util.xyz_grid([-2, 2], [-2, 2], 0, spacing=0.002)
+N = 60      # number of secondary sources
+R = 1.5     # radius of spherical/circular array
+x0, n0, a0 = sfs.array.circular(N, R)  # get secondary source positions
+
+# Signal
+L = 1024
+dirac = np.zeros(L)
+dirac[0] = 1
+signal = dirac, fs, 0
+
+# NFC-HOA order
+max_order = None
+
+# POINT SOURCE
+#xs = np.r_[1.5, 1.5, 0]  # position
+#delay, weight, sos, phaseshift = sfs.time.drivingfunction.nfchoa_25d_point(
+#                                     x0, R, xs, fs, max_order=max_order)
+#t = np.linalg.norm(xs) / sfs.defs.c
```

Comment:
We should try to avoid committing code that's commented out, see #108.

This is very hard to maintain, as exemplified here because the code is outdated (it uses `sfs.defs`).

Can you please either completely remove the code or comment it in and make an additional plot from it?